backend/api/notifications.py

```diff
@@ -14,17 +16,22 @@ def add_notification(creator, userlist, operation, project_id='',
     date = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime())
     for user in userlist:
         if user not in (creator):
-            g.notifications.insert_one({
-                '_id': str(uuid.uuid4()),
-                'creator': creator,
-                'user_id': user,
-                'project_id': project_id,
-                'operation': operation,
-                'date': date,
-                'reason': reason,
-                'active': 'true',
-                'saved_search_id': saved_search_id
-            })
+            user_object = g.user_datastore.get_user(user)
+            if user_object:
+                dict = user_object.to_dict()
+                notifications_settings = dict['notifications_settings']
+                if notifications_settings[operation] == 'true':
```

Comment:
I feel like we should use actual booleans here and not strings 'true' or 'false'. Also should use notification_settings.get(operation, False) to avoid crashes if operation is not a (valid) key in the dict.